addon/components/hyper-table-v2/manage-columns.ts

```diff
@@ -0,0 +1,91 @@
+import { A } from '@ember/array';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import TableHandler from '@upfluence/hypertable/core/handler';
+import { ColumnDefinition } from '@upfluence/hypertable/core/interfaces';
+
+type ManageColumn = {
+  definition: ColumnDefinition;
+  visible: boolean;
+};
+
+interface HyperTableV2ManageColumnsArgs {
+  handler: TableHandler;
+}
+
+export default class HyperTableV2ManageColumns extends Component<HyperTableV2ManageColumnsArgs> {
+  @tracked displayAvailableFields: boolean = false;
+  @tracked _activeColumnCategory: null | string = null;
+  @tracked _searchColumnDefinitionKeyword: null | string = null;
+
+  get _columnCategories(): string[] {
+    return this.args.handler.columnDefinitions
+      .reduce((categories: string[], columnDefinition) => {
+        if (!categories.includes(columnDefinition.category)) {
+          categories.push(columnDefinition.category);
+        }
+        return categories;
+      }, [])
+      .sort((a, b) => a.localeCompare(b));
+  }
+
+  get _orderedFilteredClusters() {
+    let fields = A(
```

Comment:
What's `A(` ?

Reply:
Oh it's ember array.
...
...
WTF Ember 🤦

Reply:
I can remove it with another name if you want, like `EmberArray` or `ArrayEmber` to be more explicit